cvat/apps/dataset_manager/default_settings.py

```diff
@@ -3,12 +3,34 @@
 # SPDX-License-Identifier: MIT
 
 import os
+import warnings
+from django.core.exceptions import ImproperlyConfigured
+
 
 DATASET_CACHE_TTL = int(os.getenv("CVAT_DATASET_CACHE_TTL", 60 * 60 * 24))
 "Base lifetime for cached exported datasets, in seconds"
 
-DATASET_CACHE_LOCK_TIMEOUT = int(os.getenv("CVAT_DATASET_CACHE_LOCK_TIMEOUT", 10))
+default_dataset_export_lock_ttl = 30
+
+DATASET_EXPORT_LOCK_TTL = int(os.getenv("CVAT_DATASET_EXPORT_LOCK_TTL",  default_dataset_export_lock_ttl))
```

Comment:
@zhiltsov-max, Since these settings will be used for backups also, do you mind if we rename them?
`EXPORT_CACHE_LOCK_TTL`, `EXPORT_CACHE_LOCK_ACQUISITION_TIMEOUT`, `EXPORT_LOCKED_RETRY_INTERVAL`

Reply:
Ok, but in this case they should probably be moved into the engine, as backups are implemented there.